finder-app/Makefile

```diff
@@ -0,0 +1,15 @@
+
+CC = gcc
+
+ifeq ($(CROSS_COMPILE), aarch64-none-linux-gnu-)
```

Comment:
@almu6129 there's a simpler way to handle this which we'll discuss in class tomorrow.